engine/engine_test.go

```diff
@@ -228,16 +232,15 @@ func TestNew(t *testing.T) {
 			false,
 		},
 	}
-
-	goamz.RetryingClient = &http.Client{
-		// force the goamz http call (to AWS to get auth details for an instance role)
-		// to return failure and hence stops auth succeeding inside CI (when we need it to fail)
-		Transport: BadTransport{},
-	}
+	ctx := context.TODO()
 
 	for _, fixture := range fixtures {
+		origconfigfunc := loadDefaultConfigFunc
```

Comment:
```suggestion
		origConfigFunc := loadDefaultConfigFunc
```
go prefers camel-case